docs/Help/FAQ.rst

```diff
@@ -0,0 +1,151 @@
+----------------------------
+
+coala stands for "COde AnaLysis Application", works well with animals and thus
+is well visualizable, it's easy to memorize.
+
+Is there corporate backing behind coala? What are your intentions?
+------------------------------------------------------------------
+
+coala was founded for fun. coala was, is and will always be free software and
+is developed mostly by students and there's no corporate interest and no CLA.
+If you want to back us, contact us on our
+`gitter channel <https://coala.io/chat>`__.
+
+What sort of analysis does coala do? What languages are supported?
+------------------------------------------------------------------
+
+A list of all analysis routines and supported languages is
+`here <https://github.com/coala/bear-docs/blob/master/README.rst#supported-languages>`__
+- fully browsable.
+
+For a top level view on what languages support what kind of analysis roughly,
+consult `this link <https://docs.google.com/spreadsheets/d/1bm63TQHndmGf3HQ33fp9UEmGKNYI7dTkjMyFIof2PqA/edit?usp=sharing>`__.
+
+There are also generic bears, which can be applied language independently on
+your code. Their capabilities and information can be seen
+`here <https://github.com/coala/bear-docs/blob/master/README.rst#all>`__.
+
+How do I get started with coala?
+--------------------------------
+
+If you're looking to get started using coala, we have a full tutorial
+`here`_
+that will teach you everything you need to know to use coala.
+
+If you're willing to contribute and become a part of our coalaian community,
+we have written a guide that will help you fix an issue on your own, just by
+following and understanding the indications
+:doc:`here <../Developers/Newcomers_Guide>`.
+It is meant for newcomers, and it does not require you to have any precedent
+knowledge regarding coala.
+
+How do I get in touch with the coala team?
+------------------------------------------
+
+We are very active on our
+`gitter channel <https://coala.io/chat>`__
+and will try to respond to any question in a matter of minutes.
+However, for a full list of how to get in touch with us, consult
+:doc:`this link <Getting_In_Touch>`.
+
+Installation is Failing! Help!
+------------------------------
+
+Don't panic!
+
+Scroll down the error log, you will probably see something like `ValueError:
+('Expected version spec in', 'appdirs ~=1.4.0', 'at', ' ~=1.4.0')` there.
+If not, `ask us! <coala.io/chat>`__
+
+If so, you're probably on a Debian with an outdated pip that doesn't support
+our version specifiers yet. You have to create a virtual environment with
+a newer pip:
+
+.. code-block:: bash
+
+  pip3 install virtualenv
+  virtualenv -p python3 ~/venvs/coala
+  . ~/venvs/coala/bin/activate
+  pip install -U pip
+  pip install coala-bears
+
+
+should do the job. You have to activate this virtualenv on every terminal
+session you want to use coala though (tip: add it to bashrc!)
+
+What are those things failing/passing on my Pull Request?
```

Comment:
we can create a devs FAQ later, lets do the move and create the FAQ in an own commit/pr?